src/data/phase/dto.type.ts

```diff
@@ -20,11 +20,21 @@ export type TaskDTO = Readonly<{
   points: number
 }>
 
+export type ChallengeContent = Readonly<{
+  challenge: string
```

Comment:
Will be better to handle a `JSX.Element` no? 
This will be more customizable than a simple text in a `div` content.

By the way, I would have named it `description`